download_data.sh

```diff
@@ -9,8 +9,13 @@ PROCESSED_DIR="$REPO/Data/Processed_Data"
 mkdir -p $PROCESSED_DIR
 
 # get transliterations
+# TODO: Update README to include steps without an Azure subscription key
```

Comment:
Leaving this as a reminder to update the README, will update in later PR if required and will remove this when done